scripts/sync-to-downstream.sh

```
@@ -1113,6 +1236,583 @@ EOF
+  local kit_targets=$6        # newline-separated
+  local templated_list=$7     # comma-joined, display only
+  local recreate_existing_pr_num=${8:-}
+  local short_sha=${sha:0:7}
+
+  local tmp_root=${TMPDIR:-/tmp}
+  local workspace
+  workspace=$(mktemp -d "$tmp_root/mergepath-sync-${consumer_name}.XXXXXX") || {
+    err "could not create workspace tmpdir"
+    return 1
+  }
+  trap "rm -rf '$workspace'" RETURN
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)